v0.3.0/Azure-NDmv4-A100/baseline.json

```diff
@@ -0,0 +1,391 @@
+  "ib-loopback/IB_write_4194304_Avg_6": 23941.39922956845,
+  "ib-loopback/IB_write_4194304_Avg_7": 23941.39922956845,
+  "ib-loopback/IB_write_512_Avg_0": 1524.7187951889875,
+  "ib-loopback/IB_write_512_Avg_1": 1524.7187951889875,
+  "ib-loopback/IB_write_512_Avg_2": 1524.7187951889875,
+  "ib-loopback/IB_write_512_Avg_3": 1524.7187951889875,
+  "ib-loopback/IB_write_512_Avg_4": 1524.7187951889875,
+  "ib-loopback/IB_write_512_Avg_5": 1524.7187951889875,
+  "ib-loopback/IB_write_512_Avg_6": 1524.7187951889875,
+  "ib-loopback/IB_write_512_Avg_7": 1524.7187951889875,
+  "ib-loopback/IB_write_524288_Avg_0": 23919.070394782873,
+  "ib-loopback/IB_write_524288_Avg_1": 23919.070394782873,
+  "ib-loopback/IB_write_524288_Avg_2": 23919.070394782873,
+  "ib-loopback/IB_write_524288_Avg_3": 23919.070394782873,
+  "ib-loopback/IB_write_524288_Avg_4": 23919.070394782873,
+  "ib-loopback/IB_write_524288_Avg_5": 23919.070394782873,
+  "ib-loopback/IB_write_524288_Avg_6": 23919.070394782873,
+  "ib-loopback/IB_write_524288_Avg_7": 23919.070394782873,
+  "ib-loopback/IB_write_65536_Avg_0": 23564.743578135713,
+  "ib-loopback/IB_write_65536_Avg_1": 23564.743578135713,
+  "ib-loopback/IB_write_65536_Avg_2": 23564.743578135713,
+  "ib-loopback/IB_write_65536_Avg_3": 23564.743578135713,
+  "ib-loopback/IB_write_65536_Avg_4": 23564.743578135713,
+  "ib-loopback/IB_write_65536_Avg_5": 23564.743578135713,
+  "ib-loopback/IB_write_65536_Avg_6": 23564.743578135713,
+  "ib-loopback/IB_write_65536_Avg_7": 23564.743578135713,
+  "ib-loopback/IB_write_8192_Avg_0": 20161.813883499373,
+  "ib-loopback/IB_write_8192_Avg_1": 20161.813883499373,
+  "ib-loopback/IB_write_8192_Avg_2": 20161.813883499373,
+  "ib-loopback/IB_write_8192_Avg_3": 20161.813883499373,
+  "ib-loopback/IB_write_8192_Avg_4": 20161.813883499373,
+  "ib-loopback/IB_write_8192_Avg_5": 20161.813883499373,
+  "ib-loopback/IB_write_8192_Avg_6": 20161.813883499373,
+  "ib-loopback/IB_write_8192_Avg_7": 20161.813883499373,
+  "ib-loopback/IB_write_8388608_Avg_0": 23923.57286780035,
+  "ib-loopback/IB_write_8388608_Avg_1": 23923.57286780035,
+  "ib-loopback/IB_write_8388608_Avg_2": 23923.57286780035,
+  "ib-loopback/IB_write_8388608_Avg_3": 23923.57286780035,
+  "ib-loopback/IB_write_8388608_Avg_4": 23923.57286780035,
+  "ib-loopback/IB_write_8388608_Avg_5": 23923.57286780035,
+  "ib-loopback/IB_write_8388608_Avg_6": 23923.57286780035,
+  "ib-loopback/IB_write_8388608_Avg_7": 23923.57286780035,
+  "kernel-launch/event_overhead": 0.0055452148625,
+  "kernel-launch/wall_overhead": 0.0094175929125,
+  "lstm_models/pytorch-lstm/steptime_train_float16": 25.7568640774,
+  "lstm_models/pytorch-lstm/steptime_train_float32": 46.7522082028,
+  "lstm_models/pytorch-lstm/throughput_train_float16": 8953.4512764025,
+  "lstm_models/pytorch-lstm/throughput_train_float32": 4893.5940230333,
+  "mem-bw/D2D_Mem_BW": 1371.0422549295124,
+  "mem-bw/D2H_Mem_BW": 25.9457922174125,
+  "mem-bw/H2D_Mem_BW": 25.895621148900002,
```

Comment:
I thought we have unified the capital letter to lowercase?

Reply:
It's v0.3 dataset, not v0.4.